applications/time_dependent/model_heat.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) 2020, Washington University in St. Louis.
```

Comment:
Update copyright note:

```
# Copyright (c) 2016-2018, The University of Texas at Austin 
# & University of California--Merced.
# Copyright (c) 2019-2022, The University of Texas at Austin 
# University of California--Merced, Washington University in St. Louis.
# Copyright (c) 2023-2024, The University of Texas at Austin 
# & University of California--Merced.
#
# All Rights reserved.
# See file COPYRIGHT for details.
#
# This file is part of the hIPPYlib library. For more information and source code
# availability see https://hippylib.github.io.
#
# hIPPYlib is free software; you can redistribute it and/or modify it under the
# terms of the GNU General Public License (as published by the Free
# Software Foundation) version 2.0 dated June 1991.
```

Reply:
Updated. Do the copyrights in `misfit.py`, `timeDependentVector`, etc. need to be updated as well?

Reply:
Yes, please